pkg/operator/ceph/object/s3-handlers.go

```diff
@@ -97,15 +116,16 @@ func (s *S3Agent) createBucket(name string, infoLogging bool) error {
 		Bucket: &name,
 	}
 
-	_, err := s.Client.CreateBucket(bucketInput)
+	_, err := s.Client.CreateBucket(context.TODO(), bucketInput)
```

Comment:
RE: `context.TODO()`: if possible, we should try to pass contexts from "above". In this case, I'd suggest that we update `createBucket()`, `CreateBucket()`, and `CreateBucketNoInfoLogging()` to take a context as input and then pass that through here.

```go
func (s *S3Agent) createBucket(ctx context.Context, name string, infoLogging bool) error {
```